torch/testing/_internal/common_methods_invocations.py

```diff
@@ -145,6 +145,104 @@ def _getattr_qual(obj, name, default=_NOTHING):
         else:
             raise
 
+# NOTE: OpInfos and the OpInfo testing pattern
+# See also: https://github.com/pytorch/pytorch/wiki/Writing-tests-in-PyTorch-1.8
+# See also: https://github.com/pytorch/pytorch/issues/54261, which tracks
+#   which OpInfos we still want to create
+#
+# An "OpInfo", described by the class and subclasses below, is a collection
+#   of metadata related to a PyTorch operator, like torch.add. This metadata
+#   can be separated into three categories:
+#
+#     1) Metadata describing the operator, like "dtypesIfCPU", which describes
```

Comment:
> Metadata describing the operator

`dtypesIfCPU` does not quite describe the operator. maybe `Metadata defining the operator support`?